wallets/eternl-wallet/src/connectWallet.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * handles the nami wallet connection
```

Comment:
Same here. Occurs multiple places.

Please check all your changes and make sure no comments, names, etc. mismatch with the current package wallet.

Reply:
understandable, was assuming this was not too much of a big deal as there was clues to stuff like this already originally: https://github.com/rosen-bridge/ui/blob/3fa552e5c23e366de56cc911ce16dcb9fa57ac8b/wallets/nami-wallet/src/index.ts#L15C16-L15C16

will correct as much as I can though 🙌